wire-ios/Wire-iOS/Sources/Authentication/Event Handlers/Initial Sync/AuthenticationInitialSyncEventHandler.swift

```diff
@@ -1,54 +0,0 @@
-//
-// Wire
-// Copyright (C) 2018 Wire Swiss GmbH
-//
-// This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with this program. If not, see http://www.gnu.org/licenses/.
-//
-
-import Foundation
-import WireCommonComponents
-import WireUtilities
-
-/**
- * Handles the initial sync event.
- */
-
-final class AuthenticationInitialSyncEventHandler: NSObject, AuthenticationEventHandler {
-
-    weak var statusProvider: AuthenticationStatusProvider?
-
-    func handleEvent(currentStep: AuthenticationFlowStep, context: Void) -> [AuthenticationCoordinatorAction]? {
-        // Skip email/password prompt for @fastLogin automation
-        guard AutomationHelper.sharedHelper.automationEmailCredentials == nil else {
```

Comment:
question: is this needed by QA to run automationtests?

Reply:
I don't think this had any effect all because when we continue passed the guard we execute the same actions.